MAUI/DataGrid/Unbound-Column.md

```diff
@@ -0,0 +1,201 @@
+<td> / </td>
+</tr>
+<tr>
+<td> Power</td>
+<td> ^ </td>
+</tr>
+<tr>
+<td> Mod</td>
+<td> % </td>
+</tr>
+<tr>
+<td> Greater Than</td>
+<td> < </td>
+</tr>
+<tr>
+<td> Less than</td>
+<td> > </td>
+</tr>
+<tr>
+<td> Equal</td>
+<td> = </td>
+</tr>
+<tr>
+<td> GreaterThanOrEqual</td>
+<td> >= </td>
+</tr>
+<tr>
+<td> LessThanOrEqual </td>
+<td> <= </td>
+</tr>
+</table>
+
+Logical operation
+
+<table>
+<tr>
+<th> Logical operations </th>
+<th> Operators </th>
+</tr>
+<tr>
+<td> AND </td>
+<td> (char)135 </td>
+</tr>
+<tr>
+<td> OR  </td>
+<td> (char)136 </td>
+</tr>
+<tr>
+<td> NOT </td>
+<td> (char)137 </td>
+</tr>
+<tr>
+</table>
+
+{% tabs %}
+{% highlight xaml %}
+
+ <syncfusion:SfDataGrid x:Name="datagrid"    
+                  ColumnWidthMode="Fill"
+                  AutoGenerateColumnsMode="None"
+                  ItemsSource="{Binding OrderInfoCollection1}">
+     <syncfusion:SfDataGrid.Columns>
+         <syncfusion:DataGridUnboundColumn 
+                            MappingName="DiscountPrice"
+                            HeaderText="SUM"/>
+     </syncfusion:SfDataGrid.Columns>
+ </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight C# %}
+    datagrid.QueryUnboundColumnValue += DataGrid_QueryUnboundColumnValue;
+    
+    private void DataGrid_QueryUnboundColumnValue(object? sender, DataGridUnboundColumnEventArgs e)
+    {
+        if (e.UnboundAction == DataGridUnboundActions.QueryData)
+            {
+                var Price1 = Convert.ToInt16(e.Record!.GetType().GetProperty("Price1")?.GetValue(e.Record));
+                var Price2 = Convert.ToInt16(e.Record!.GetType().GetProperty("Price2")?.GetValue(e.Record));
+                var amount = Price1 * Price2;
+                e.Value = amount;
+            }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+# Using QueryUnboundColumnValue event
+
+The QueryUnboundColumnValue event is fired when value for the unbound column is quired. It provides information about the cell that triggered this event. So, you can set the desired value for the grid cells of the unbound column. This event is triggered with the DataGridUnboundEventArgs.
+
+The DataGridUnboundColumnEventArgs provides the following properties:
+
+***Column:*** Gets DataGridColumn of the cell that triggers this event.
+***OriginalSender:*** Gets the data grid raising event.
+***Record:*** Gets the underlying row data
+***UnboundAction:*** Defines the action for triggering event.
+***Value:*** Gets or Sets the value for DataGridUnboundCOlumn cell based on unboundAction.
+
+# Populate the data for the unbound column by handling.
+
+QureyUnboundColumnValue event which allows customizing the value of the DataGridUnboundColumn. DataGridUnboundCOlumnEventArgs exposes Value property, by which you can set the value for the grid cells of the unbound column based on the UnboundAction.
+
+Refer to the following code example in which data for the unbound column is populated by handling the queryUnboundColumnValue event:
```

Comment:
Populate the data for the unbound column by handling `QueryUnboundColumnValue` event which allows customizing the value of the `DataGridUnboundColumn`. `DataGridUnboundColumnEventArgs` exposes `Value` property, by which you can set the value for the grid cells of the unbound column based on the `UnboundAction`.

Refer to the following code example in which data for the unbound column is populated by handling the `QueryUnboundColumnValue` event: